lib/index.js

```diff
@@ -41,12 +41,17 @@ Server.prototype._createServer = function() {
     return;
   }
 
+  // connections
+  self._connections = [];
+
   var server = net.createServer(function(connection) { //'connection' listener
     new Connection(connection, self, {
       version: self.protocolVersion,
       authKey: self.authKey,
       protocol: self.protocol
     });
+    // push to connections array
+    self._connections.push(connection);
```

Comment:
You need to remove to be able to remove the connection too. You probably need a listener on the connection for that.
You also probably don't want to loop over all connections every time one is closed, so you probably want a hash map here and therefore an id on the connections (an incremental integer would do the trick)